Orange/widgets/regression/owsgdregression.py

```diff
@@ -115,7 +114,10 @@ def __init__(self):
         power_t = gui.doubleSpin(box, self, "power_t", 0.0, 10.0, step=0.01)
         form.addRow("Power t:", power_t)
 
-
+        scroll = QtGui.QScrollArea()
+        scroll.setWidget(box)
+        scroll.setWidgetResizable(True)
+        scroll.setFixedHeight(400)
```

Comment:
Please see the [Coding style section](https://github.com/biolab/orange3/blob/master/CONTRIBUTING.md#coding-style) of the guidelines for how to set static Qt properties.